### DISCUSSION TITLE
Gmail Oauth, functional?

### DISCUSSION THREAD

**Discussion by bagstoper** [moralmunky/Home-Assistant-Mail-And-Packages#974] [Q&A] [answered]
There are references to it in other issues about app passwords not working but I don't see any options to provide a client id and client secret for gmail oauth. 

Upon adding the integration I am only given options to enter IMAP connection details. 
![image](https://github.com/user-attachments/assets/2989dfb1-1b5b-44fd-890f-f375601fc54a)

I don't see an oauth branch in HACS, I don't have an option to choose gmail oauth like in the pics in this post
https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/discussions/726

It is mentioned in the wiki but there is nothing that says what to do with them once created. 
https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki/Configuration-and-Email-Settings/#oauth-setup

**Comment by firstof9** [comment]
There's no Oauth config at this time.

**Reply by bagstoper** [reply]
Thanks for the quick confirmation. The last link I added made me think I was missing something.

**Reply by firstof9** [reply, marked as answer]
New feature soon to be added will resolve this: https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/pull/1207
